### PR TITLE
Change normalize to make it export friendly

### PR DESCRIPTION
**Pull request by larryliu0820**
As titled. torch.export doesn't work with `_get_kernel` since it looks into attributes of a class. Here we are assigning a kernel manually for torch.export.

### CODE REVIEW

**Comment by NicolasHug** [comment]
Thanks for the PR @larryliu0820 and sorry for the late reply, I was away last week.

Support for `torch.export` is still an ongoing effort for the v2 transforms and we still lack a bit of clarity on that front. Can you please share a snippet of what you're trying to enable exactly? E.g. are you trying to export the class? The functional? The kernel? FYI, earlier this year I summarized what is currently supported by torchvision in terms of compile/export ([internal link](https://fb.workplace.com/groups/374285425240042/permalink/402879965713921/)).
